src/calculateRentalCost.js

```diff
@@ -20,7 +20,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result;
+  if (days >= 7) {
+    result = days * 40 - 50;
```

Comment:
you can write like this
```suggestion
    return days * 40 - 50;
```